config/config.go

```diff
@@ -41,6 +41,7 @@ type Config struct {
 	DebugListener                   string         `yaml:"debug-listener"`
 	CertPathTLS                     string         `yaml:"tls-cert"`
 	KeyPathTLS                      string         `yaml:"tls-key"`
+	StartupChecks                   *listFlag      `yaml:"startup-checks"`
```

Comment:
I believe we should think about extending this functionality in the future into continuous checks, and in that case this name would become deprecated very quick. Can you think about a better name that doesn't suggest that it's currently only for the startup?

Maybe HealthcheckHooks? StatusHooks? StatusChecks?

Reply:
I am not sure if I would change to a generic name, because we might just drop this in favor for a more generic solution as we discussed.

Reply:
dropping may cause unnecessary maintenance cost. I see that the interface may change, and that's why we're suggesting to mark it experimental, but since there is a good chance that we keep it, it is worth making it more generic with this anticipation in this case.

Reply:
We decided to go with StatusChecks